### PR TITLE
Aarch64

### PR DESCRIPTION
**Pull request by nerohmot**
Checklist
* [X] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [X] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by nerohmot** [comment]
changed aarch support from azure to default as requested

**Comment by ccordoba12** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ source:
 
 build:
   skip: true  # [win]
-  number: 0
+  number: 1
```
Comment (ccordoba12) [inline recipe/meta.yaml]: I think there's no need to increase the build number, so this should be reverted.

Comment (ccordoba12) [reply recipe/meta.yaml]: @conda-forge/core, is it necessary to increase the build number to add support for aarch64?

Comment (jjhelmus) [reply recipe/meta.yaml]: No, the existing non-aarch64 packages do not need to be rebuilt so there is no need to change the build number.  That said, bumping the build number is fine, it just means a few new packages are created that are not absolutely needed.

Comment (nerohmot) [reply recipe/meta.yaml]: ok, I will de-bump and submit a new pr.

**Comment by nerohmot** [comment]
@conda-forge-admin, please rerender

**Comment by ccordoba12** [comment]
I think this is ready to be merged after @nerohmot reopens this PR.

@minrk, could you take a look at this one?

**Pull request closed without merging**